charts/osbuilder/Chart.yaml

```diff
@@ -8,5 +8,5 @@ maintainers:
 - name: Ettore Di Giacinto
   email: mudler@kairos.io
 
-version: 0.5.1
+version: 0.5.2
 appVersion: "v0.5.1"
```

Comment:
The "app" version hasn't changed (we still run the `v0.5.1` version of the osbuilder) so I'm only bumping the helm chart version